homeassistant/components/samsungtv/bridge.py

```diff
@@ -319,8 +319,9 @@ async def async_get_app_list(self) -> dict[str, str] | None:
     def _get_app_list(self) -> dict[str, str] | None:
         """Get installed app list."""
         if self._app_list is None and (remote := self._get_remote()):
-            with contextlib.suppress(WebSocketTimeoutException):
+            with contextlib.suppress(TypeError, WebSocketTimeoutException):
                 raw_app_list: list[dict[str, str]] = remote.app_list()
+                LOGGER.debug("Received app list: %s", raw_app_list)
```

Comment:
This doesn't work in latest release.

Reply:
This is added just to add information in case app_list causes more issues...